app/src/main/res/layout/activity_create_profile.xml

```diff
@@ -1,15 +1,87 @@
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:gravity="center"
-    tools:context=".Create_Profile.CreateProfileActivity">
+    android:gravity="center_horizontal"
+    tools:context=".Create_Profile.CreateProfileActivity"
+    android:background="@drawable/bg_for_createprofile">
+<RelativeLayout
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:layout_gravity="center_horizontal"
+    >
+    <Switch
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:switchPadding="0dp"
+        android:text="Bluetooth"
+        android:layout_marginTop="60dp"
+        android:gravity="center_horizontal"
+        android:id="@+id/bluetooth"
+        android:paddingRight="80dp"
+        android:textSize="25sp" />
 
+    <Switch
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="120dp"
+        android:switchPadding="0dp"
+        android:gravity="center_horizontal"
+        android:id="@+id/airplane"
+
+        android:paddingRight="60dp"
+        android:text="Airplane Mode"
+        android:textSize="25sp" />
     <TextView
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="180dp"
+        android:textSize="25sp"
+        android:gravity="center"
+        android:textColor="@color/black"
+        android:text="Alarm"/>
+    <TimePicker
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:text="Create Profile"/>
+        android:id="@+id/time_picker"
+        android:layout_marginTop="220dp"
+        />
+    <RadioGroup
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:id="@+id/radio_group"
+        android:gravity="center"
+        android:layout_marginTop="600dp">
+        <TextView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="Mode"
+            android:textSize="22sp"/>
+
+        <RadioButton
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="General"
+            android:textSize="18sp"/>
+
+        <RadioButton
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="Mute"
+            android:textSize="18sp"/>
+
+    </RadioGroup>
+
+`   <Button
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:text="Save Profile"
+    android:layout_marginTop="700dp"
```

Comment:
Do not hardcode layout_marginTop. While using RelativeLayout use android:layout_below and android:layout_above to spe